differences-between-specimin-and-perses.md

```diff
@@ -59,4 +59,128 @@ Differences:
 * Specimin removes the `VERSION_INFORMATION` field, which apparently doesn't contribute to the error (why does Perses keep it?)
 * Specimin replaces the body of `getAllVersionInformation()` with `throw new Error();`, but Perses leaves it as-is
 * Perses moves the contents of the target method `method()` to an initializer block
-* Perses removes the `final` from the declaration of the `versionInfo` local
\ No newline at end of file
+* Perses removes the `final` from the declaration of the `versionInfo` local
+
+## NA-323
+
+Specimin Output:
+```
+package com.cogvio.time;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.time.ZoneId;
+
+public final class MutableClock extends Clock {
+
+    private Instant instant;
+
+    private ZoneId zone;
+
+    private MutableClock(final Instant instant, final ZoneId zone) {
+        this.setZone(zone);
+        this.setInstant(instant);
+    }
+
+    public void setInstant(final Instant instant) {
+        throw new Error();
+    }
+
+    public void setZone(final ZoneId zone) {
+        throw new Error();
+    }
+
+    public ZoneId getZone() {
+        throw new Error();
+    }
+
+    public MutableClock withZone(final ZoneId newZone) {
+        throw new Error();
+    }
+
+    public Instant instant() {
+        throw new Error();
+    }
+}
+
+```
+
+Perses Output:
+```
+package com.cogvio.time;
+
+import java.time.Instant;
+import java.time.ZoneId;
+
+class MutableClock {
+  Instant instant;
+  ZoneId zone;
+
+  MutableClock() {}
+}
+
+```
+Line counts using same formatting:
+* Specimin: 27
+* Perses: 8
+
+Differences:
+* Perses doesn't include clock
+* Perses removes "public final" before "class" in class declaration and removes "extends"
+* Perses removes "private" from variable declarations
+* Perses removes arguments for MutableClock()
+* Specimin replaces the body of the setters and getters with "throw new Error();" while perses removes them entirely
+
+## Running Perses on Specimin Output
+
+We also ran Perses on the output of Specimin for two cases where the output was
+preserved by Specimin, and the output was a single file.
```

Comment:
Do you have the Perses run times for these experiments? It would be great to show that Specimin + Perses produces better/equal output but is much faster; this shows better/equal output.